src/App.js

```diff
@@ -189,11 +192,24 @@ export class App extends React.Component {
                   <PrivateRoute exact path='/request-access/:id' component={RequestAccess} />
                 </div>
               </Switch>
-            {this.state.userSession.isAuthenticated
-              && <Footer />}
-          </div>
-        </BrowserRouter>
-      </UserContext.Provider>
+                {this.state.userSession.isAuthenticated
```

Comment:
Ah, so aside from adding the "ToastContainer" here, I cleaned up the indentation which is why there's some changes here.